docs/blockchain/contracts/nft-contract/nft-actions/mkfctofr.a.experimental.md

```diff
@@ -0,0 +1,74 @@
+---
+title: 'mkfctofr.a'
+order: 40
+
+---
+
+# mkfctofr.a
+
+Make a buy offer on an Uniq factory.
+
+## Technical Behavior
+
+The offer should be done at the time which is in the range of trading window of the factory.
+
+The action stores the offer to `fctoffer.a` table with the specified arguments. The new offer ID is read from `next.fctofr` table whose `value` field is then incremented.
+
+`eosio.nftram` pays RAM usage.
+
+The offered price will be transferred to `eosio.nftofr` account and will be kept until the offer is either accepted or cancelled.
+
+An account will not be able to make offers on the same factories that they already made.
+
+An account will not be able to make more offers on factories than `max_active_offer_per_user` of `offercfg.a`, which records the global configurations.
+
+`price` should be no less than min_price of `offercfg.a` and also should be no less than minimum_resell_price of the factory.
```

Comment:
Should be
`price` should be no less than `min_price` of `offercfg.a` and also should be no less than `minimum_resell_price` of the factory.

Reply:
updated